go/education.html

```diff
@@ -0,0 +1,132 @@
+    
+  </ul>
+  <p class="highlight">See our other great apps for calendars, diagrams, forums, and more: <a href="http://apps.sandstorm.io">App Market</a>.</p>
+</section>
+
+<section id="why-it">
+  <h2>Here's why the IT team will love it</h2>
+    <h3>Sandstorm comes with <script type="text/javascript">document.write(appCount)</script>+ great apps for every project. Instructors and students can also package their own apps that anyone can then install.</h3>
+  <ul>
+    
+    <li>
+      <h3>On-prem install for education privacy</h3>
+      <p>Run Sandstorm on hardware that you control. Everything users do on a Sandstorm server stays safely within your institution. Enjoy free dynamic DNS and SSL.</p>
+      
+    <li>
+      <h3>Self-service software</h3>
+      <p>Sandstorm provides simple web access to every user at the institution. When you approve Sandstorm, users do what they want safely within Sandstorm, and they file fewer support tickets.</p>
+
+    <li>
+      <h3>Single-sign-on and web access</h3>
+      <p>Once you enable login via Google, GitHub, or email, every app automatically uses that login provider. LDAP and SAML are to arrive soon.</p>
+    
+    <li>
+      <h3>Custom apps can use the built-in access control</h3>
+      <p>Academic departments need custom apps. As those apps get older, Sandstorm helps you sleep well at night. Sandstorm automatically mitigates against <a href="https://docs.sandstorm.io/en/latest/using/security-non-events/">95% of app security issues, according to our analysis.</a></p>
+      
+    <li>
+      <h3>Access control like Google Docs</h3>
+      <p>Sandstorm makes all data private by default. Users can share access through a familiar interface.</p>
+      
+    <li>
+      <h3>Automatic app downscaling</h3>
+      <p>When a user closes a browser tab to an app, Sandstorm waits a few minutes, then terminates the processes. When the user visits the app again, Sandstorm starts it automatically.</p>
+      
+    <li>
+      <h3>Centralized backups and data migration</h3>
+      <p>Your Sandstorm server stores all data for all apps and all the databases they use. When you take a backup of it, you have everything. </p>
+      
+    <li>
+      <h3>Open source software</h3>
+      <p>Sandstorm is open source software enjoyed by thousands of individuals at non-profits and companies every day. The company behind Sandstorm is building Sandstorm for Work, a paid offering adding more SSO integration.</p>
+    
+  </ul>
+  <p class="highlight">Learn more about <a href="/how-it-works">Sandstorm's security model</a>.</p>
+</section>
+
+<section id="UNBC">
+<h2>See how UNBC uses Sandstorm</h2>
+  <div class="content">
+  <div class="text">
+  <p>Grant Potter at the University of Northern British Columbia runs a server supporting 70+ faculty and staff and writes tutorials to help people learn Piwik and other tools. He also uses Sandstorm apps to take meeting minutes. </p>

```

Comment:
It feels a little funny to mention "meeting minutes" which sounds like a very small thing, but I don't have a better suggestion.